google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -525,24 +525,38 @@ public boolean exists(BlobSourceOption... options) {
   }
 
   /**
-   * Fetches current blob's latest information. Returns {@code null} if the blob does not exist.
+   * Fetches the latest blob properties. Returns {@code null} if the blob no longer exists.
    *
-   * <p>Example of getting the blob's latest information, if its generation does not match the
-   * {@link Blob#getGeneration()} value, otherwise a {@link StorageException} is thrown.
+   * <p>{@code options} parameter can contain the preconditions. E.g. user may need to get the blob
```

Comment:
E.g. --> For example,

Google style prohibits Latin abbreviations

Reply:
E.g. is widely used, perhaps we need to collect such things and do some code clean up.